app/com/gu/contentapi/sanity/CrosswordsIndexingTest.scala

```diff
@@ -0,0 +1,33 @@
+package com.gu.contentapi.sanity
+
+import org.joda.time.{DateTime}
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatest.{Matchers, FlatSpec}
+import play.api.libs.json.Json
+
+class CrosswordsIndexingTest extends FlatSpec with Matchers with ScalaFutures with IntegrationPatience {
+
+  "A new Crossword" should "be indexed every day" taggedAs(LowPriorityTest, PRODTest) in {
+
+    handleException {
+      val now = DateTime.now
+      val isAfterChristmas = now.getMonthOfYear == 12 && (now.getDayOfMonth == 26 || now.getDayOfMonth == 27)
+      val twentyFiveHoursAgo = now.minusHours(25)
+      assume(!isAfterChristmas, "Cancelling as Crosswords are not always published over the Christmas period")
+
+      val httpRequest = requestHost("search?tag=type/crossword&order-by=newest").get
+      whenReady(httpRequest) { result =>
+        assume(result.status == 200, "Service is down")
+        val json = Json.parse(result.body)
+        val newestItem = (json \ "response" \ "results")(0)
```

Comment:
This line will throw an IndexOutOfBounds exception if no results are returned, whcih isn't a very informative/helpful message.
A better approach would be to use headOption, which gets the first item and wraps it in an Option so it won't throw an error if the results are empty, it'll just return None:
 val newestItemOpt = (json \ "response" \ "results").headOption

Then you can check if the item is defined, with some appropriate failure message:
newstItemOpt.isDefined should be(true) 

Then you can either get the newstItem using newsetItemOpt.get or to be really clean you can use:
newestItemOpt foreach { newestItem =>
... rest of the test code goes here.
}